### PR TITLE
bump vault to 0.9.4, adding multi platform support

### PR DESCRIPTION
**Pull request by chrishoffman**

### CODE REVIEW

File: library/vault [hunk 1]
```diff
@@ -1,4 +1,8 @@
-# maintainer: Jeff Mitchell <jeff@hashicorp.com> (@jefferai)
+Maintainers: Chris Hoffman <choffan@hashicorp.com> (@hoffmancs),
+             Jeff Mitchell <jeff@hashicorp.com> (@jefferai)
+GitRepo: https://github.com/hashicorp/docker-vault.git
 
-0.9.3: git://github.com/hashicorp/docker-vault@00d387db577abc178e2b0740ed11955d1e0aa860 0.X
-latest: git://github.com/hashicorp/docker-vault@00d387db577abc178e2b0740ed11955d1e0aa860 0.X
+Tags: 1.9.4, latest
```
Comment (yosifkit) [inline library/vault]: `1.9.4`?! :open_mouth:

Comment (chrishoffman) [reply library/vault]: shoot, nope

Comment (chrishoffman) [reply library/vault]: Thanks for catching that, pushed!

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index 4e1fc81..0909db4 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1 +1,4 @@
 vault:latest @ amd64
+vault:latest @ arm32v6
+vault:latest @ arm64v8
+vault:latest @ i386
diff --git a/_bashbrew-list b/_bashbrew-list
index 57e3e66..e7b6177 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -1,2 +1,2 @@
-vault:0.9.3
+vault:0.9.4
 vault:latest
diff --git a/vault_latest/Dockerfile b/vault_latest/Dockerfile
index 5daf0b8..0e308e0 100644
--- a/vault_latest/Dockerfile
+++ b/vault_latest/Dockerfile
@@ -1,8 +1,7 @@
 FROM alpine:3.7
-MAINTAINER Jeff Mitchell <jeff@hashicorp.com> (@jefferai)
 
 # This is the release of Vault to pull in.
-ENV VAULT_VERSION=0.9.3
+ENV VAULT_VERSION=0.9.4
 
 # Create a vault user and group first so the IDs get set the same way,
 # even as the rest of this may change over time.
@@ -10,16 +9,25 @@ RUN addgroup vault && \
     adduser -S -G vault vault
 
 # Set up certificates, our base tools, and Vault.
-RUN apk add --no-cache ca-certificates gnupg openssl libcap su-exec dumb-init && \
+RUN set -eux; \
+    apk add --no-cache ca-certificates gnupg openssl libcap su-exec dumb-init && \
+    apkArch="$(apk --print-arch)"; \
+    case "$apkArch" in \
+        armhf) ARCH='arm' ;; \
+        aarch64) ARCH='arm64' ;; \
+        x86_64) ARCH='amd64' ;; \
+        x86) ARCH='386' ;; \ 
+        *) echo >&2 "error: unsupported architecture: $apkArch"; exit 1 ;; \
+    esac && \
     gpg --keyserver pgp.mit.edu --recv-keys 91A6E7F85D05C65630BEF18951852D87348FFC4C && \
     mkdir -p /tmp/build && \
     cd /tmp/build && \
-    wget https://releases.hashicorp.com/vault/${VAULT_VERSION}/vault_${VAULT_VERSION}_linux_amd64.zip && \
+    wget https://releases.hashicorp.com/vault/${VAULT_VERSION}/vault_${VAULT_VERSION}_linux_${ARCH}.zip && \
     wget https://releases.hashicorp.com/vault/${VAULT_VERSION}/vault_${VAULT_VERSION}_SHA256SUMS && \
     wget https://releases.hashicorp.com/vault/${VAULT_VERSION}/vault_${VAULT_VERSION}_SHA256SUMS.sig && \
     gpg --batch --verify vault_${VAULT_VERSION}_SHA256SUMS.sig vault_${VAULT_VERSION}_SHA256SUMS && \
-    grep vault_${VAULT_VERSION}_linux_amd64.zip vault_${VAULT_VERSION}_SHA256SUMS | sha256sum -c && \
-    unzip -d /bin vault_${VAULT_VERSION}_linux_amd64.zip && \
+    grep vault_${VAULT_VERSION}_linux_${ARCH}.zip vault_${VAULT_VERSION}_SHA256SUMS | sha256sum -c && \
+    unzip -d /bin vault_${VAULT_VERSION}_linux_${ARCH}.zip && \
     cd /tmp && \
     rm -rf /tmp/build && \
     apk del gnupg openssl && \
```

</details>

**Comment by yosifkit** [comment]
Build test of #4031; 612940e9; `amd64` (`vault`):

```console

$ bashbrew build vault:0.9.4
Using bashbrew/cache:b84abf2f8c65c6adc072cfd6a608a3604865cacb54c521772ea37ef573fc168a (vault:0.9.4)
Tagging vault:0.9.4
Tagging vault:latest

$ test/run.sh vault:0.9.4
testing vault:0.9.4
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**